podman.md

```diff
@@ -44,10 +44,10 @@ podman exec -it HibernateTestingMariaDB mysql -U hreact -phreact
 
 * Server
 ```
-sudo podman run --rm -it -e LICENSE=accept --privileged=true --name HibernateTestingDB2 -e DBNAME=hreact -e DB2INSTANCE=hreact -e DB2INST1_PASSWORD=hreact -e PERSISTENT_HOME=false -e ARCHIVE_LOGS=false -e AUTOCONFIG=false -p 50000:50000 ibmcom/db2:11.5.0.0a
```

Comment:
I don't know why, but when I remove `sudo` the DB2 container crashes on my machine. I'd leave it be and move on

Reply:
It's a bit annoying to have some machine starting with sudo and some without because they won't appear if you run `podman ps` or `sudo podman ps`. I would prefer if our instructions were consistent.

Reply:
I think `sudo` is required depending on what level of permissions you normally have without `sudo`. The DB2 container requires `--privileged` so that's probably why `sudo` is also needed.

Reply:
Seems to always work fine (without `sudo`) using `docker` fwiw

Reply:
right, it works fine for @DavideD as well. I don't know why but on my specific machine this container seems very problematic - it crashes often with either option, but I've only been able to boot it successfully with `sudo`